website/projects/scimaps/src/app/core/models/book.model.ts

```diff
@@ -0,0 +1,9 @@
+export interface Book {
+  title: string;
+  author: string;
+  publisher: string;
+  abstract: string;
+  pdfLink: string;
+  amazonLink: string;
+  id: number;
```

Comment:
id: number; => slug: string;

Reply:
We also need to work out which fields are needed for the index and which for the detail page. The index.json will contain _only_ the data needed for listings, but not the whole shebang. Also, abstract will likely be Markdown.

Reply:
Okay, I'll change that field to markdown.